setup.py

```diff
@@ -3,6 +3,7 @@
 from bump_version import get_current_version
 
 PYTORCH_REQUIRES = ['torch>=1.6.0']
+JAX_REQUIRES = PYTORCH_REQUIRES + ['jaxlib>=0.1.75', 'jax>=0.2.26', 'flax>=0.3.6']
 SPACY_REQUIRES = ['spacy>=3.2.0', 'spacy-transformers>=1.1.0']
 ALL_REQUIRES = (PYTORCH_REQUIRES +
                 SPACY_REQUIRES)
```

Comment:
`ALL_REQUIRES` should also include `JAX_REQUIRES`.

Reply:
Added.